.github/workflows/actions/setup-node/action.yml

```diff
@@ -3,16 +3,17 @@ name: Setup
 inputs:
   use-cache:
     description: Restore global `~/.npm` cache
-    required: false
+    default: 'true'
+    required: true
 
 runs:
   using: composite
 
   steps:
     - name: Setup Node.js
-      uses: actions/setup-node@v3.6.0
+      uses: actions/setup-node@v3.7.0
       id: setup-node
 
       with:
-        cache: ${{ inputs.use-cache != 'false' && 'npm' || '' }}
+        cache: ${{ inputs.use-cache == 'true' && 'npm' || '' }}
```

Comment:
This was [the fix](https://github.com/alphagov/govuk-frontend/pull/3954/commits/b8165f9c53bfb8303cdc506df2e424ac03d89980) that unblocked https://github.com/alphagov/govuk-frontend/pull/3954/commits/65965b2968e678cdadbe3ca8f8f486c815b4c560 but speeds up Windows cache restoration times

Reply:
Moved to https://github.com/alphagov/govuk-frontend/pull/4014